server/fishtest/views.py

```diff
@@ -1164,82 +1207,78 @@ def tests_modify(request):
         request.session.flash("Please login")
         return HTTPFound(location=request.route_url("login"))
 
-    if "num-games" in request.POST:
-        run = request.rundb.get_run(request.POST["run"])
-        before = del_tasks(run)
-
-        now = datetime.now(timezone.utc)
-        if (
-            "start_time" not in run
-            or (now - run["start_time"].replace(tzinfo=timezone.utc)).days > 30
-        ):
-            request.session.flash("Run too old to be modified", "error")
-            return home(request)
+    run = request.rundb.get_run(request.POST["run"])
+    if run is None:
+        request.session.flash("No run with this id", "error")
+        return home(request)
 
-        if not can_modify_run(request, run):
-            request.session.flash("Unable to modify another user's run!", "error")
-            return home(request)
+    validate_modify(request, run)
 
-        num_games = int(request.POST["num-games"])
-        if (
-            num_games > run["args"]["num_games"]
-            and "sprt" not in run["args"]
-            and "spsa" not in run["args"]
-        ):
-            request.session.flash(
-                "Unable to modify number of games in a fixed game test!", "error"
-            )
-            return home(request)
+    if not can_modify_run(request, run):
+        request.session.flash("Unable to modify another user's run!", "error")
+        return home(request)
 
-        max_games = 3200000
-        if num_games > max_games:
-            request.session.flash(
-                "Number of games must be <= " + str(max_games), "error"
-            )
-            return home(request)
+    is_approver = request.has_permission("approve_run")
+    if not is_approver and (
+        (
+            run["args"]["throughput"] != int(request.POST["throughput"])
```

Comment:
`!=` should probably be changed to `<` here, since we can allow e.g. decreasing throughput if it was set to above 100 previously

Reply:
> `!=` should probably be changed to `<` here, since we can allow e.g. decreasing throughput if it was set to above 100 previously

Please look at the code again, != here is only the first condition to check if before != new i.e. check if changed